code/modules/mob/living/carbon/human/monkey/monkey.dm

```diff
@@ -76,6 +76,7 @@ GLOBAL_DATUM(the_one_and_only_punpun, /mob/living/carbon/human/species/monkey/pu
 		equip_to_slot_or_del(new relic_hat, ITEM_SLOT_HEAD)
 	if(relic_mask)
 		equip_to_slot_or_del(new relic_mask, ITEM_SLOT_MASK)
+	equip_to_slot_or_del(new /obj/item/clothing/under/suit/waiter(src), ITEM_SLOT_ICLOTHING)
```

Comment:
Do we want pun pun to start with this or have bartenders buy the suit from their vendor?

Reply:
starting with it is better, you can remove it from Pun Pun if you don't like it